packages/next-swc/crates/next-core/src/next_shared/resolve.rs

```diff
@@ -147,36 +138,50 @@ impl Issue for InvalidImportModuleIssue {
 
     #[turbo_tasks::function]
     async fn description(&self) -> Result<Vc<OptionStyledString>> {
-        let message = match self.import_type {
-            InvalidImportType::ClientOnly => {
-                "'client-only' cannot be imported from a Server Component module. It should only \
-                 be used from a Client Component."
-            }
-            InvalidImportType::ServerOnly => {
-                "'server-only' cannot be imported from a Client Component module. It should only \
-                 be used from a Server Component."
-            }
-        };
+        let raw_context = &*self.file_path.await?;
+
+        let mut messages = self.messages.clone();
+        messages.push("\n".to_string());
+
+        //[TODO]: how do we get the import trace?
+        messages.push(format!(
```

Comment:
1. we don't get import trace to display
2. also this place doesn't know `styled-jsx` import from the `<styled..` tag